Emilia-Capital/aaa-option-optimizer
# use popover for values

## Context
The columns don't fit on a screen if options have very large values. This PR uses the new 'popover' built in browser modal to show the value in, and ads some css to style it. 

## Summary

This PR can be summarized in the following changelog entry:

* Move option value to a popover. 

## Relevant technical choices:

* Inspired by this: https://twitter.com/jdevalk/status/1780511519941300474, I've used the browser built in popover for the modal. 
* Style is in an scss file, not sure if this is the preferred method in this project
* I've also excluded the assets from loading on other pages than the plugin page itself, as a suggestion. 

## Test instructions
- Install a plugin with very large option values, like WooCommerce. Load the admin page in the current version: you'll see the columns won't fit on the page. 
- Install this PR, values are now replaced with buttons. On clicking them, a modal opens with the value. 

#### Relevant test scenarios

## UI changes


## Documentation


## Quality assurance

* [v ] I have tested this code to the best of my abilities.

Fixes #
